src/injector.js

```diff
@@ -0,0 +1,103 @@
+const puma = require('pumascript');
+const http = require('http');
+const fs = require('fs');
+
+
+let errorsLogs = [];
+
+
+function readDependencyJsonFile(dir) {
+    try {
+        return JSON.parse(fs.readFileSync(dir, 'utf8'));
+    } catch (e) {
+        console.log('The file entered does not exist');
+    }
+    return undefined;
+}
+
+
+function readUrl(options, callback) {
+    var req = http.get(options, function (res) {
+        var bodyChunks = [];
+        res.on('data', function (chunk) {
+            bodyChunks.push(chunk);
+        }).on('end', function () {
+            var body = Buffer.concat(bodyChunks);
+            callback(body);
+        })
+    });
+    req.on('error', function (e) {
+        console.log('ERROR: ' + e.message);
+    });
+}
+
+
+
+function createFile(dir, fileName, data) {
+    fs.writeFileSync(dir + '/' + fileName, data);   
+}
+
+//Error object
+function Error(url) {
+    this.url = '' || url;
+    this.listErrors = [];
+}
+Error.prototype.addError = function (line, colum, componentGeneratesError) {
+    this.listErrors.push({
+        line, colum, componentGeneratesError
+    })
+}
+
+
+function testPuma(dataUrl, url) {
+    console.info('********** Entering ', url, '****************************');
+    var result = puma.evalPuma(dataUrl, 'test');
+    let error = new Error();
+
+    error.listErrors = [];
+    if (result.success !== undefined) {
+        if (result.success) {
+            console.info('++++++++++++++ Successful injection ++++++++++++++++++');
+            error.url = url;
+        } else {
+            console.error(`Error when interpreting the file, puma does not support any internal components.
```

Comment:
I think indentations spaces would pollute console output.

Reply:
In my opinion the sangria does not contaminate the output of the console.